com.sap.cloud.lm.sl.cf.core/src/test/resources/com/sap/cloud/lm/sl/cf/core/cf/clients/service-01.json

```diff
@@ -1,5 +1,5 @@
 {
-  "service": {
+  "initialService": {
```

Comment:
In testing the appropiate words for inputs are "actual" and "expected".
So, maybe you culd use actualService ?